strawberry_django_plus/optimizer.py

```diff
@@ -79,6 +80,7 @@
 else:
     _relation_fields = (models.ManyToManyField, ManyToManyRel, ManyToOneRel)
 _sentinel = object()
+_annotate_placeholder = "______annotate_placeholder______"
```

Comment:
Maybe just `__annotated_placeholder__` would be enough? Any reason to be using 4 underscores?

Reply:
I can change to `__annotated_placeholder__`, it should be more than enough to avoid clashes. Also, if you have a suggestion on how to avoid this, please LMK. Unfortunately, I needed this workaround because field names aren't bind at class declaration moment.